pkg/provisioner/reconciler/claim-reconciler/resourcehandlers.go

```diff
@@ -238,3 +238,49 @@ func deleteObjectBucket(ob *v1alpha1.ObjectBucket, ic *internalClient) error {
 	}
 	return nil
 }
+
+func updateObjectBucketClaimPhase(ic *internalClient, obc *v1alpha1.ObjectBucketClaim, phase v1alpha1.ObjectBucketClaimStatusPhase, retryInterval, retryTimeout time.Duration) (*v1alpha1.ObjectBucketClaim, error) {
```

Comment:
can all Phase updates for OBs and OBCs, be combined into a more generic func? Similar to `createUntilDefaultTimeout`?